code/game/machinery/biogenerator.dm

```diff
@@ -257,20 +257,23 @@
 	category="Misc."
 
 /datum/biogen_recipe/misc/pest
-	id="pest"
-	name="Insecticide"
-	reagent=INSECTICIDE
+	id="pestspray"
+	name="Pest Spray"
 	cost=35
-	amount_per_unit=10
-	other_amounts=list(5)
+	result=/obj/item/weapon/plantspray/pestspray/
```

Comment:
Is this the phased out single use pest spray?

Reply:
Currently, it's six uses

Reply:
Bad then. The pest spray didn't harm bugmen (it's the reason we swapped to Insecticide). Restore it to the actual insecticide.

Reply:
I'm pretty sure this can't harm bugmen, it works the same way as before???????

Reply:
That's precisely why we changed it to insecticide, it supposedly harmed bugs (in the tray) but didn't harm insectoids. See #28873

Reply:
You could also make the weedkiller a chemical instead of a single use item, but the problem stands either way: Why use the weedkiller 6-use item when the perfectly fine, available at roundstart and with infinite uses minihoe/fork do the same?

Reply:
I just replaced the bugspray with an actual chemical. I did not have a reason to touch the unused but still "implemented" bay botany chemicals.

Reply:
Okay, this should fix everything if you want to look